source/vstd/arithmetic/power2.rs

```diff
@@ -239,4 +237,168 @@ pub proof fn lemma2_to64()
     ) by(compute_only);
 }
 
+/// Mask with low n bits set.
+pub open spec fn mask(n: nat) -> nat {
```

Comment:
Should we be more explicit here? `low_bits_mask`?

Should all these be in a separate `mask.rs` file?

Reply:
I'd be inclined to put them in `bits.rs`, since it seems like the primary use case is reasoning about bit-manipulation.  And yes, I would use something more explicit, since there are many kinds of mask one might want.  I think `low_bits_mask` or `lsb_mask` could work.